casual/casual-caller/src/main/java/se/laz/casual/connection/caller/ConnectionFactoryEntryStore.java

```diff
@@ -7,6 +7,7 @@
 package se.laz.casual.connection.caller;
 
 import se.laz.casual.connection.caller.config.ConfigurationService;
+import se.laz.casual.connection.caller.topologychanged.TopologyChangedHandler;
```

Comment:
Unnecessary import?

Reply:
Nope, sonar and IntelliJ are fine with it

Reply:
Oh, I see now the handler moved to another package and now needs to be imported. That explains the import only diff here and for the tests